driver-portal/src/Tests/ApiIntegrationTestBase.cs

```diff
@@ -10,7 +10,7 @@ public abstract class ApiIntegrationTestBase
         protected HttpClient _client { get; }
         protected readonly IConfiguration _configuration;
         protected const string CASE_API_BASE = "/api/Cases";
-        protected const string DRIVER_API_BASE = "/api/Drivers";
+        protected const string DRIVER_API_BASE = "/api/Driver";
```

Comment:
We'll have to discuss this - whether we use Driver, Account, or Profile to represent the logged in user.  No change required at the moment though.